scripts/superdesk/upload/styles/upload.less

```diff
@@ -580,87 +580,93 @@ media uploader
 edit image modal
 ======================================
 */
-.change-image {
-    &__metadata {
-        position: absolute;
-        top: 0;
-        right: 0;
-        bottom: 225px;
-        width: 335px;
-        overflow-y: auto;
-        background-color: #f0f0f0;
-        border-left: 1px solid #dfdfdf;
-        .box-sizing(border-box);
-        padding: 15px;
-        > div {
-            margin-bottom: 20px;
-        }
-        label {
-            font-size: 11px;
-            text-transform: uppercase;
-            color: @grayLight;
-        }
-    }
+.modal-body__flex {
+	display: flex;
+	flex-direction: column;
+	padding: 0;
+	overflow-y: hidden !important;
```

Comment:
then as I told you in another PR, it's better to play with selector precision than `important!`